doc/samtools-ampliconstats.1

```diff
@@ -0,0 +1,280 @@
+'\" t
+.TH samtools-ampliconstats 1 "6 December 2019" "samtools-1.10" "Bioinformatics tools"
+.SH NAME
+samtools ampliconstats \- produces statistics from amplicon sequencing alignment file
+.\"
+.\" Copyright (C) 2020 Genome Research Ltd.
+.\"
+.\" Author: James Bonfield <jkb@sanger.ac.uk>
+.\"
+.\" Permission is hereby granted, free of charge, to any person obtaining a
+.\" copy of this software and associated documentation files (the "Software"),
+.\" to deal in the Software without restriction, including without limitation
+.\" the rights to use, copy, modify, merge, publish, distribute, sublicense,
+.\" and/or sell copies of the Software, and to permit persons to whom the
+.\" Software is furnished to do so, subject to the following conditions:
+.\"
+.\" The above copyright notice and this permission notice shall be included in
+.\" all copies or substantial portions of the Software.
+.\"
+.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+.\" THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+.\" LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+.\" FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+.\" DEALINGS IN THE SOFTWARE.
+.
+.\" For code blocks and examples (cf groff's Ultrix-specific man macros)
+.de EX
+
+.  in +\\$1
+.  nf
+.  ft CR
+..
+.de EE
+.  ft
+.  fi
+.  in
+
+..
+.
+.SH SYNOPSIS
+.PP
+samtools ampliconstats
+.RI [ options ]
+.IR primers.bed
+.IR in.sam | in.bam | in.cram ...
+
+.SH DESCRIPTION
+.PP
+samtools ampliconstats collects statistics from one or more input
+alignment files and produces tables in text format.  The output can be
+visualized graphically using plot-ampliconstats.
+
+The alignment files should have previously been clipped of primer
+sequence, for example by "samtools ampliconclip" and the sites of
+these primers should be specified as a bed file in the arguments.
+Each amplicon must be present in the bed file with one or more LEFT
+primers (direction "+") followed by one or more RIGHT primers.  For
+example:
+
+.EX 2
+MN908947.3  1875  1897  nCoV-2019_7_LEFT        nCoV-2019_1  +
```

Comment:
Strictly this is an invalid BED file.  Column 5 should be a score, i.e. a number.  The V2 `nCoV-2019.bed` got it right, the V3 one was wrong.